### ISSUE TITLE
nod explorer incorrectly applying format codes in some strings

### ISSUE THREAD

**Issue by ezio416** [openplanet-nl/openplanet-issues#492]
I'm looking through my replays in the nod explorer and formatting codes are being applied when I feel they should be ignored entirely and instead show the raw string. Example of what I mean (this map name starts with `$19F`, and since the Windows path contains a backslash, it seems it's reading `\$19F` as a color code it should format):
![Screenshot 2024-05-29 202737](https://github.com/openplanet-nl/issues/assets/37951898/6bd7e2fe-482e-4a32-a87d-33416119e8da)

**Comment by XertroV** [comment]
lol yeah that's a bug

**Comment by XertroV** [comment]
Not sure exactly how to solve it though. @ezio416 does 'copy' work as expected?

**Comment by ezio416** [comment]
Yes, the copy button works as expected, it's just a visual thing

**Comment by ezio416** [comment]
Here is that replay file which may help in testing
[$19FMazio $G003_EzioTM_2023-10-16_16-46-10(01'27''588).Replay.zip](https://github.com/openplanet-nl/issues/files/15493334/19FMazio.G003_EzioTM_2023-10-16_16-46-10.01.27.588.Replay.zip)

**Comment by codecat** [comment]
I could pretty easily fix this, but it would be at the cost of Nod Explorer performance. I'm not sure it's worth it tbh.
